src/calc.c

```diff
@@ -310,8 +316,23 @@ static char* calc_preprocess_input(Mode* sw, const char* input)
     GError *error = NULL;
     CALCModePrivateData* pd = (CALCModePrivateData*)mode_get_private_data(sw);
 
-    const gchar* const argv[] = { "qalc", "+u8", "-s", "update_exchange_rates 1days", input, NULL };
+    const gchar* const default_args[] = { "+u8", "-s", "update_exchange_rates 1days", input, NULL };
+    int len = sizeof(default_args) / sizeof(const gchar*);
```

Comment:
You can use [G_N_ELEMENTS()](https://developer.gnome.org/glib/stable/glib-Standard-Macros.html#G-N-ELEMENTS:CAPS) here instead.